docs/optimization.rst

```diff
@@ -0,0 +1,276 @@
+best satisfy your throughput and latency requirements.
+
+Framework-Specific Optimization
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The inference server has several optimization setting that apply only
+to subset of the supported model frameworks. These optimization
+settings are controlled by the model configuration :ref:`optimization
+policy <section-optimization-policy>`.
+
+One especially powerful optimization that we will explore here is to
+use :ref:`section-optimization-policy-tensorrt` in conjunction with a
+TensorFlow or ONNX model.
+
+ONNX with TensorRT Optimization
+...............................
+
+As an example of TensorRT optimization applied to an ONNX model, we
+will use the ONNX DenseNet model that you can obtain by following the
+:ref:`section-quickstart`. As a baseline we use perf\_client to
+determine the performance of the model using a `basic model
+configuration that does not enable any performance features
+<https://github.com/NVIDIA/tensorrt-inference-server/blob/master/docs/examples/model_repository/densenet_onnx/config.pbtxt>`_::
+
+  $ perf_client -m densenet_onnx --percentile=95 --concurrency-range 1:4
+  ...
+  Inferences/Second vs. Client p95 Batch Latency
+  Concurrency: 1, 113.2 infer/sec, latency 8939 usec
+  Concurrency: 2, 138.2 infer/sec, latency 14548 usec
+  Concurrency: 3, 137.2 infer/sec, latency 21947 usec
+  Concurrency: 4, 136.8 infer/sec, latency 29661 usec
+
+To enable TensorRT optimization for the model: stop the inference
+server, add the following lines to the end of the model configuration
+file, and then restart the inference server::
+
+  optimization { execution_accelerators {
+    gpu_execution_accelerator : [ { name : "tensorrt" } ]
+  }}
+
+As the inference server starts you should check the console output and
+wait until the server prints the "Staring endpoints" message. ONNX
+model loading can be significantly slower when TensorRT optimization
+is enabled.  Now run perf\_client using the same options as for the
+baseline::
+
+  $ perf_client -m densenet_onnx --percentile=95 --concurrency-range 1:4
+  ...
+  Inferences/Second vs. Client p95 Batch Latency
+  Concurrency: 1, 190.6 infer/sec, latency 5384 usec
+  Concurrency: 2, 273.8 infer/sec, latency 7347 usec
+  Concurrency: 3, 272.2 infer/sec, latency 11046 usec
+  Concurrency: 4, 266.8 infer/sec, latency 15089 usec
+
+The TensorRT optimization provided 2x throughput inprovement while
+cutting latency in half. The benefit provided by TensorRT will vary
+based on the model, but in general it can provide significant
+performance improvement.
+
+TensorFlow with TensorRT Optimization
+.....................................
+
+TensorRT optimization applied to a TensorFlow model works similarly to
+TensorRT and ONNX described above. To enable TensorRT optimization you set the model configuration appropriately. For TensorRT optimization of TensorFlow models that are several options that you can enable, including selection the precision. For example::
```

Comment:
To enable TensorRT optimization you must set the model configuration appropriately.